mozci/task.py

```diff
@@ -195,6 +196,19 @@ def artifacts(self):
         """List the artifacts that were uploaded by this task."""
         return [artifact["name"] for artifact in list_artifacts(self.id)]
 
+    @property
+    def autoclassify(self):
```

Comment:
I'd make this a top-level function that, for now, takes `task` as input. It's the same in practice, but I prefer logically separating it from the Task class (it is a function that operates on tasks, not a property of the task or a method of the task)

Reply:
I kind of feel the same way about the `Push` class, but that's out of scope for now. Maybe we can make a new `classify.py` or `regressions.py` module to start housing this stuff. But leaving it in `task.py` is fine for now.

Reply:
Done